ambassador/ambassador/envoy/v2/v2listener.py

```diff
@@ -509,6 +510,38 @@ def __init__(self, config: 'V2Config', listener: IRListener) -> None:
                     'format': 'ACCESS [%START_TIME%] \"%REQ(:METHOD)% %REQ(X-ENVOY-ORIGINAL-PATH?:PATH)% %PROTOCOL%\" %RESPONSE_CODE% %RESPONSE_FLAGS% %BYTES_RECEIVED% %BYTES_SENT% %DURATION% %RESP(X-ENVOY-UPSTREAM-SERVICE-TIME)% \"%REQ(X-FORWARDED-FOR)%\" \"%REQ(USER-AGENT)%\" \"%REQ(X-REQUEST-ID)%\" \"%REQ(:AUTHORITY)%\" \"%UPSTREAM_HOST%\"\n'
                 }
             } ]
+             # Use sane access log spec in JSON
```

Comment:
There's a stray space before that `#` that makes the comment stick out a bit... (Sorry this is the most pedantic comment :D )